romana_setup.md

```diff
@@ -77,15 +77,18 @@ The `uninstall` action is used to delete hosts created during install (`aws` and
 
 ## Ansible Options
 
-In some cases, you may wish to pass additional options to Ansible for the installation.
+In some cases, you may wish to override the value of variables that Ansible uses during installation.
 
 You can do this by either:
 * Editing values in `romana-install/group_vars` files
 * Providing ansible options at the end of the `romana-setup` command
 
-All items on the command-line after the `install` or `uninstall` verb are passed to Ansible.
+All items on the command-line after the `install` or `uninstall` verb are passed to Ansible. [Examples](#examples) can be seen below.
 
-An example [example](#examples) can be seen below.
+Some noteworthy variables are:
+* `host_cidr`: The IP/Prefix for hosts created during installation. Default: `192.168.99.0/24`. (Applies to `aws` and `vagrant`.). For AWS, the prefix should be 16 or higher.
```

Comment:
I think we should say "prefix size should be 16 or higher", since the prefix on its own is just the 192.168.99 part.

Same in line 90.